### ISSUE TITLE
Wrong release group detected.

### ISSUE THREAD

**Issue by duramato** [guessit-io/guessit#547]
The resolution gets wrongfully detected as the release group. There's no release group present in this release name.

```
For: Panorama.15-05-2018.Web-DL.540p.H264.AAC.Subs.mp4
GuessIt found: {
    "title": "Panorama",
    "date": "2018-05-15",
    "format": "WEB-DL",
    "release_group": "540p",
    "video_codec": "h264",
    "audio_codec": "AAC",
    "subtitle_language": "und",
    "container": "mp4",
    "mimetype": "video/mp4",
    "type": "episode"
}
```
```
+-------------------------------------------------------+
+                   GuessIt 2.1.4                       +
+-------------------------------------------------------+
+                   Rebulk 0.9.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
```

**Comment by VeNoMouS** [comment]
@duramato strange this only does it via the app, not via the module itself

```
>>> guessit('Panorama.15-05-2018.Web-DL.540p.H264.AAC.Subs.mp4')
MatchesDict([('title', 'Panorama'), ('date', datetime.date(2018, 5, 15)), ('source', 'Web'), ('video_codec', 'H.264'), ('audio_codec', 'AAC'), ('release_group', 'Subs'), ('container', 'mp4'), ('mimetype', 'video/mp4'), ('type', 'episode')])
>>> 
```

**Comment by Toilal** [comment]
@VeNoMouS I have refactored the way configuration files are loaded to make it consistent between module and CLI, it this now give the same results as the CLI after https://github.com/guessit-io/guessit/commit/74195f745ca514b1bbb564d6ef28ae3d96d23dda.

You may also have a look to this commit because it enhance configuration files loading and allow `advanced_config` to be passed as an option inside guessit api module.

**Comment by VeNoMouS** [comment]
<3 your work @Toilal  :)

**Comment by ratoaq2** [comment]
You can still define 540p as a valid resolution in the advanced options
https://github.com/guessit-io/guessit/blob/develop/docs/configuration.rst

https://github.com/guessit-io/guessit/blob/develop/guessit/config/options.json/#L329

**Comment by ratoaq2** [comment]
Fixed by #a164b6a2c33dc1e2f153ae944079f259b41ca1db
